build/__dotNetCoreSdkVersions.sh

```diff
@@ -10,3 +10,4 @@ DOT_NET_50_SDK_VERSION='5.0.408'
 DOT_NET_60_SDK_VERSION='6.0.401'
 DOT_NET_60_RTM_SDK_VERSION='6-0-100-rtm-21527-11'
 DOT_NET_70_SDK_VERSION='7.0.100-rc.2.22477.23'
+RUNTIME_VERSIONS=("1.0" "1.1" "2.0" "2.1" "2.2" "3.0" "3.1" "5.0" "6.0" "7.0")
```

Comment:
Adding lists to shell output created these `RUNTIME_VERSIONS` lists. I realize that they will overwrite each other because they are the same name, but they are never used and so I think that it should be ok